.gitignore

```diff
@@ -1,16 +1,8 @@
-*.pyc
-*.pyo
-*~
+*.egg-info
+*.py[co]
 *.sw[po]
+*~
 .DS_Store
-.coverage
-.installed.cfg
-MANIFEST
```

Comment:
I thought the same, but I was unable to generate this file with both Python 2 and Python 3. Do you have an example command that results in this file being generated? I tried:

```
$ python3 setup.py sdist bdist_wheel
$ python2 setup.py sdist bdist_wheel
```

Is it only with a specific Python version?